lib/usd/ui/CMakeLists.txt

```diff
@@ -27,35 +27,9 @@ add_library(${PROJECT_NAME} SHARED)
 # -----------------------------------------------------------------------------
 target_sources(${PROJECT_NAME} 
     PRIVATE
-        cmds/USDImportDialogCmd.cpp
-        factories/TreeModelFactory.cpp
         images/ui.qrc
-        views/ItemDelegate.cpp
-        views/TreeItem.cpp
-        views/TreeModel.cpp
-        views/USDImportDialog.cpp
-        views/USDImportDialog.ui
-        views/USDQtUtil.cpp
 )
```

Comment:
I understand the principle here, but I think we should weigh the cost vs benefit of having a single cmake file vs many in sub-dirs for such a small project like this UI one. My personal opinion is that it was much simpler and easier to understand and edit as a single file. For example both the cmds/factories sub-dirs each only have a single. That seems such an overhead to have a separate cmakefile for each.

Reply:
Hey @seando-adsk ,

I can see your point when a project is small (e.g mayaUsdUI) with not many source files. I agree that it seems like an overhead, however the advantage of having CMakeLists.txt for each subdirectory starts to show as the project grows. We also follow the similar discipline inside Maya code base as well.

Reply:
Sure I can see that, but in this particular case this project will never grow that big. I don't see advantage to having cmake file that has a single source file, such as: lib/usd/ui/factories/CMakeLists.txt

Reply:
Internally we discussed this "ui" folder and I thought we decided that since it was such a small project we would remove all those sub-folders views/cmds/factories and just have all the sources in one folder (except for demo which is a separate project). Therefore we don't really need these new ui/xxx/cmake files as they would just be removed anyways.

Reply:
@seando-adsk Yes this is addressed as part of my post here https://github.com/Autodesk/maya-usd/pull/388#issuecomment-607543513

See this commit for chanes 9d9404a6